provider/provider_test.go

```diff
@@ -0,0 +1,46 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/pulumi/providertest"
+)
+
+func TestExamples(t *testing.T) {
+	t.Run("cluster", func(t *testing.T) {
+		test(t, "../examples/cluster").Run(t)
+	})
+}
+
+func test(t *testing.T, dir string, opts ...providertest.Option) *providertest.ProviderTest {
```

Comment:
I don't see where `t` is used.
```suggestion
func test(dir string, opts ...providertest.Option) *providertest.ProviderTest {
```